src/core/theme/colors.ts

```diff
@@ -112,8 +112,8 @@ export const gradients = {
   whiteBaseNegative: `linear-gradient(-180deg, ${alphaHex(0.1)(
     colors.whiteBase,
   )} 0%, ${alphaHex(0)(colors.whiteBase)} 100%)`,
-  depthLight26: `linear-gradient(0deg, ${colors.depthLight26} 0%, ${colors.whiteBase} 100%)`,
-  highlightLight45ToHighlightLight50: `linear-gradient(0deg, ${colors.highlightLight45}, ${colors.highlightLight50})`,
+  depthLight2: `linear-gradient(0deg, ${colors.depthLight2} 0%, ${colors.whiteBase} 100%)`,
+  highlightLight2TohighlightLight3: `linear-gradient(0deg, ${colors.highlightLight2}, ${colors.highlightLight3})`,
```

Comment:
If following the previous naming highlightLight2To**h**ighlightLight3 should be highlightLight2To**H**ighlightLight3.

Reply:
One additional letter should be capital.

Reply:
Fixed.